app/Features.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import Image from "next/image";
+
+export default function Features() {
+  return (
+    <section>
+      {/* This is for the Features header */}
+      <div>
+        <div className="flex flex-col items-center justify-center mt-15 lg:mt-16">
+          <div>
+            <p className="w-44 font-sans text-xl text-center text-black tracking-widest sm:text-4xl lg:text-5xl">Features</p>
+          </div>
+          <div className="bg-[#191970] w-44 h-1 sm:w-96 lg:w-5/12"></div>
+        </div>
+      </div>
+
+      {/* This is for the Features images and content */}
+    <div className="lg:flex lg:items-aligned lg:justify-evenly lg:mt-16">
+      <div className="flex items-aligned justify-evenly mt-7 sm:mt-14 lg:flex-col lg:justify-center ">
+        <div> 
+          <Image 
+            alt="Mockup 2" 
+            className="w-28 sm:w-56 lg:w-64 lg:items-aligned lg:ml-3" 
+            height={226} 
+            src="/mockup2.png"
+            width={111.51} 
+          /> 
+        </div>
+        <div className="custom-width-157 self-center"> 
```

Comment:
what is `custom-width-157`? doesnt look like its a tailwind CSS class name, its not even on the `globals.css`.

remove if not used

Reply:
use `w-40` nalang guro @caspted - when designin using figma use the closest value in tailwind lang siguro